pkg/storage/pebble.go

```diff
@@ -144,9 +144,9 @@ var UseExciseForSnapshots = settings.RegisterBoolSetting(
 var IngestSplitEnabled = settings.RegisterBoolSetting(
 	settings.SystemOnly,
 	"storage.ingest_split.enabled",
-	"set to true to use ingest-time splitting to lower write-amplification (experimental)",
+	"set to false to disable ingest-time splitting to lower write-amplification (experimental)",
```

Comment:
nit: it currently sounds like _disabling_ ingest-time splitting is the experimental option. We could probably reword this as "set to false to disable experimental ingest-time splitting that lowers write-amp", or drop the experimental altogether seeing as if it's defaulting to true, we should be able to stand behind it by the time 24.1 comes out.

Reply:
Done.